standardization/src/main/scala/za/co/absa/enceladus/standardization/CmdConfig.scala

```diff
@@ -158,6 +161,31 @@ object CmdConfig {
       config.copy(folderPrefix = Some(value))).text("Adds a folder prefix before the date tokens")
 
     help("help").text("prints this usage text")
+
+    private def processCobolCmdOptions(): Unit = {
+      var copybookLocation: String = ""
+      var isXcom = false
+      opt[String]("copybook").optional().action((value, config) => {
+        copybookLocation = value
+        config.copy(cobolOptions = Some(CobolOptions(copybookLocation, isXcom)))
+      }).text("Path to a copybook for COBOL data format")
+        .validate(value =>
+          if (rawFormat.isDefined && rawFormat.get.equalsIgnoreCase("cobol"))
+            success
+          else
+            failure("The --copybook option is supported only for COBOL data format")
+        )
+
+      opt[Boolean]("is-xcom").optional().action((value, config) => {
+        isXcom = value
+        config.copy(cobolOptions = Some(CobolOptions(copybookLocation, isXcom)))
```

Comment:
wouldn't this line as well as the line 170 profit from being at the end of this method and just once?